tests/python/topi/python/test_topi_batch_matmul.py

```diff
@@ -67,10 +67,13 @@ def check_device(device, ctx):
 
 @tvm.testing.uses_gpu
 def test_batch_matmul():
-    verify_batch_matmul(1, 16, 16, 32)
-    verify_batch_matmul(5, 16, 16, 32)
-    verify_batch_matmul(5, 16, 20, 32)
-    verify_batch_matmul(30, 16, 20, 32)
+    verify_batch_matmul(1, 1, 16, 16, 32)
+    verify_batch_matmul(5, 5, 16, 16, 32)
+    verify_batch_matmul(5, 5, 16, 20, 32)
+    verify_batch_matmul(30, 30, 16, 20, 32)
+    # Test batch broadcasting.
+    verify_batch_matmul(1, 5, 16, 16, 32)
+    verify_batch_matmul(5, 1, 16, 16, 32)
```

Comment:
Consider adding a test that ensures the incorrect use case is handled as expected, e.g. xb=2, yb=5.

Reply:
Its unusual to add a test that intentionally fails to the topi testing suite, it doesnt look any there are any other examples of that. We could instead add it to the relay testing of batch_matmul if youd like.

Reply:
Just commented as a consideration. I generally like to be sure things fails as I expect but don't feel strongly enough about it to ask for a change to status quo.